modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -497,6 +495,7 @@ export default {
       this.setFocusedWidget(this.widget._id, this.areaId);
     }
 
+    this.setTinyWidgetContainer();
```

Comment:
Hardcoded for test purpose, probably should be re computed when container size change.